setup.cfg

```diff
@@ -4,7 +4,7 @@ universal = 1
 [flake8]
 exclude = .tox,migrations,doc/*,shoop_tests/*
 max-line-length = 120
-max-complexity = 10
+max-complexity = 12
```

Comment:
To be honest, because I couldn't figure out an elegant way to split [link_to_parent()](https://github.com/shoopio/shoop/pull/51/files#diff-c24dd7ffdcb4bf401b37669c23303384R313) into multiple functions. :|